plugin.js

```diff
@@ -67,7 +67,7 @@ class WasmPackPlugin {
   }
 
   _compile() {
-    info('ℹ️  Compiling your crate...\n');
+    info(`ℹ️  Compiling your crate ${this.isDebug ? 'in development mode' : ''}...\n`);
```

Comment:
What about:

```suggestion
    info(`ℹ️  Compiling your crate in ${this.isDebug ? 'development' : 'release'} mode...\n`);
```

Reply:
Works for me! I think it's possible that if the `wasm-pack build` default changes in the future to something other than "release" when not specified, this statement will no longer be correct. You have more context than I do about the status of `wasm-pack` and so on, though, so this looks like a good change to me!